ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/tracker/CompletionResponse.scala

```diff
@@ -70,22 +70,18 @@ object CompletionResponse {
     response match {
```

Comment:
Who depends on this method? Shouldn't we deprecate this?

Reply:
Good point. 
Canton is the dependent that actually needs this.

I will open a new PR once this is merged, which deprecates this usage.